### PR TITLE
Rename provisioner config struct

### PR DESCRIPTION
**Pull request by humblec**
Signed-off-by: Humble Chirammal <hchiramm@redhat.com>

### CODE REVIEW

**Comment by humblec** [comment]
@rootfs @jsafrane PTAL .

**Comment by resouer** [comment]
@k8s-bot ok to test

File: pkg/volume/glusterfs/glusterfs.go [hunk 1]
```diff
@@ -368,7 +368,7 @@ func (plugin *glusterfsPlugin) newProvisionerInternal(options volume.VolumeOptio
 	}, nil
 }
 
-type provisioningConfig struct {
+type provisionerConfig struct {
```
Comment (resouer) [inline pkg/volume/glusterfs/glusterfs.go]: Sorry, but could you please clarify why need this modification?

Comment (humblec) [inline pkg/volume/glusterfs/glusterfs.go]: @resouer just as a better name for the struct.

**Comment by humblec** [comment]
@jsafrane @rootfs Can you please take a look at this PR ?

**Comment by humblec** [comment]
@saad-ali can you please review this ? :)

**Pull request merged**